compatibility-suite/tests/Model/PactPath.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace PhpPactTest\CompatibilitySuite\Model;
+
+use PhpPactTest\CompatibilitySuite\Constant\Path;
+
+class PactPath
+{
+    public const PROVIDER = 'p';
+
+    public function __construct(
+        private string $consumer = 'c'
+    ) {
+    }
+
+    public function getConsumer(): string
+    {
+        return $this->consumer;
+    }
+
+    public function getProvider(): string
+    {
+        return self::PROVIDER;
+    }
```

Comment:
Can you please choose a consistent way of defining?

I was thinking private readonly strings for both provider and consumer.

Reply:
Removed the method `getProvider`. Now only use the constant.

We didn't remove php 8.0 yet so the keyword `readonly` is not supported. The property `$consumer` is read only but without the keyword `readonly`.